IntegrationTests/Assets/APITests/WinBackOfferAPITests.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using UnityEngine;
+
+namespace DefaultNamespace
+{
+    public class WinBackOfferAPITests : MonoBehaviour
+    {
+        private void Start()
+        {
+            Purchases.WinBackOffer winBackOffer = new Purchases.WinBackOffer(null);
+            string identifier = winBackOffer.Identifier;
+            float price = winBackOffer.Price;
+            string priceString = winBackOffer.PriceString;
+            int cycles = winBackOffer.Cycles;
+            string period = winBackOffer.Period;
+            string periodUnit = winBackOffer.PeriodUnit;
```

Comment:
This is likely a bad positioning we had before, but I wonder if we should try to reuse the existing [Period](https://github.com/RevenueCat/purchases-unity/blob/5450caee72ce0532875367590d6ea1d85f70724c/RevenueCat/Scripts/SubscriptionOption.cs#L218) and [PeriodUnit](https://github.com/RevenueCat/purchases-unity/blob/5450caee72ce0532875367590d6ea1d85f70724c/RevenueCat/Scripts/SubscriptionOption.cs#L252) types

Reply:
Yeah, that's a great callout! The values are here like this to match what we're doing in purchase-unity's [Discount](https://github.com/RevenueCat/purchases-unity/blob/main/RevenueCat/Scripts/Discount.cs) class, and because it matches the interface used in the hybrids (which keep them separate because that's what the Discount class does in the other hybrids as well)